eclipse-install-plugins

```diff
@@ -24,6 +24,15 @@ if [ -z "$DISTRIB_ID" ]; then
   DISTRIB_ID=$ID
 fi
 
+if [ -n "$ECLIPSE_HOME" ] || [ $OS = "Windows_NT" ]; then
```

Comment:
Die check voor ECLIPSE_HOME is hier trouwens ook niet meer nodig, dat kan enkel een check voor Windows zijn.